tools/unix/version.sh

```diff
@@ -42,8 +42,17 @@ function android_code {
   echo "${cutYear//./}$(printf %02d "$COUNT")"
 }
 
-function git_hash {
-  git describe --match="" --always --abbrev=8 --dirty
+function qt_int_version {
+  # yy_MM_dd
+  # yy - year
+  # MM - month
+  # dd - day
+  local cutYear=${DATE:2}
+  echo "${cutYear//./}"
+}
+
+function qt_version {
+  echo "$DATE-$COUNT ($(git describe --match="" --always --abbrev=8 --dirty))"
```

Comment:
???

Reply:
local GIT_HASH=$(git describe --match="" --always --abbrev=8 --dirty)

Reply:
I have changed the format for Qt version to be consistent with Android/iOS:

- `2022.08.21-3-ad79d64e-Darwin` (instead of `2022.08.21-3 (ad79d64e)`)
- `2022.08.19-7-FDroid`
- `2022.08.19-7`